observability/power_monitoring/power-monitoring-release-notes.adoc

```diff
@@ -15,6 +15,27 @@ These release notes track the development of {PM-title} in the {product-title}.
 
 For an overview of the {PM-operator}, see xref:../power_monitoring/power-monitoring-overview.adoc#power-monitoring-about-power-monitoring_power-monitoring-overview[About {PM-shortname}].
 
+[id="power-monitoring-release-notes-0-4"]
+== {PM-shortname-c} 0.4 (Technology Preview)
+
+This release includes the following version updates:
+
+* {PM-kepler} 0.7.12
+* {PM-operator} 0.15.0
+
+
+[id="power-monitoring-0-4-features"]
+=== Features
+
+* With this release, FIPS mode is now enabled for {PM-title}. When installed on an {OCP} cluster in FIPS mode, {PM-operator} ensures compatibility without affecting the cluster’s FIPS support status.
+
+* Power monitoring is now migrated from Cloud Platform as a Service (CPasS) to Konflux improving platform stability, performance, and automation. No action is required from customers.
```

Comment:
the change from CPaaS to konflux is only in the way the product is built. this does not "platform stability, performance, and automation"